lib/service/kubernetes.go

```diff
@@ -242,6 +242,7 @@ func (process *TeleportProcess) initKubernetesService(log *logrus.Entry, conn *C
 			Component:                     teleport.ComponentKube,
 			StaticLabels:                  cfg.Kube.StaticLabels,
 			DynamicLabels:                 dynLabels,
+			CloudLabels:                   cloudLabels,
```

Comment:
Why not just take `process.cloudLabels` here instead of passing it as an argument?